src/components/atoms/button.js

```diff
@@ -0,0 +1,45 @@
+import React, { Children } from "react"
+
+const Button = React.forwardRef(
```

Comment:
Would argue that we might as well convert the button to TS now. What do you think?

Reply:
Definitely! :)